info/gradlew.bat

```diff
@@ -0,0 +1,104 @@
+@rem
```

Comment:
Please address my previous comment about gradle wrapper scripts.

Reply:
Yeah, it was a mistake adding this file

Reply:
Fixed